README.md

```diff
@@ -0,0 +1,30 @@
+
+![](./images/MB.png)
+
+# System Overview.
+
+The Motherboard has seven identical slots for ![](https://latex.codecogs.com/gif.latex?35mm%20%5Ctimes%2040mm) modules which
+are connected to a single CPU module slot which has up to seventy general purpose
+inputs and outputs (GPIO), as well as ![](https://latex.codecogs.com/gif.latex?SPI), ![](https://latex.codecogs.com/gif.latex?I%5E%7B2%7DC), and ![](https://latex.codecogs.com/gif.latex?USB) communication buses.
+The Motherboard also supplies each module with a direct battery connection
+and additionally regulated ![](https://latex.codecogs.com/gif.latex?5V) and ![](https://latex.codecogs.com/gif.latex?3.3V) lines.
+
+## Block Diagram
+
+![](./images/Connection-Diagram.png)
+
+![](./images/Supply-Diagram.png)
+
+## License
+
+This work is licensed under a [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode)
+
+**Attribution** — You must give [appropriate credit](/license/credit.md), provide a link to the license, and
+[indicate if changes were made](https://wiki.creativecommons.org/License_Versions#Modifications_and_adaptations_must_be_marked_as_such).
+You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
+
+**NonCommercial** — You may not use the material for [commercial purposes]().
+
+**ShareAlike** — If you remix, transform, or build upon the material, you must distribute your contributions under the [same license](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode) as the original.
+
+![](https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png)
```

Comment:
I miss a simple note clarifying that these are `Eagle` designs. `KiCad` uses the same file extensions.